x-pack/plugin/snapshot-repo-test-kit/src/internalClusterTest/java/org/elasticsearch/repositories/blobstore/testkit/RepositoryAnalysisFailureIT.java

```diff
@@ -445,8 +478,18 @@ public boolean acceptsEmptyRegister() {
         }
     }
 
-    private void analyseRepository(RepositoryAnalyzeAction.Request request) {
-        client().execute(RepositoryAnalyzeAction.INSTANCE, request).actionGet(5L, TimeUnit.MINUTES);
+    private RepositoryVerificationException analyseRepository(RepositoryAnalyzeAction.Request request) {
```

Comment:
Nit: I wonder whether the method name could be more indicative that it is expected to fail?
```suggestion
    private RepositoryVerificationException analyseRepositoryAndExpectFailure(RepositoryAnalyzeAction.Request request) {
```

Reply:
Fair point, see 68f8328bcb4d.